array/sort_by.mbt

```diff
@@ -129,21 +124,18 @@ fn try_bubble_sort_by[T](
   cmp : (T, T) -> Int,
   ~max_tries : Int = 8
 ) -> Bool {
-  let mut tries = 0
-  for i = 1; i < arr.length(); i = i + 1 {
-    let mut sorted = true
-    for j = i; j > 0 && cmp(arr[j - 1], arr[j]) > 0; j = j - 1 {
-      sorted = false
-      arr.swap(j, j - 1)
-    }
-    if not(sorted) {
-      tries += 1
-      if tries > max_tries {
-        return false
+  let mut sorted = false
+  for t = 1; t < max_tries && not(sorted); t = t + 1 {
```

Comment:
This modification is wrong. We should avoid sorting the entire `arr` using bubble sort. The original version will abort the bubble sort if it meets `max_tries` unsorted items.